test/testdata/java_book_names.baseline

```diff
@@ -14,17 +14,20 @@
 
 package com.google.example.library.v1;
 
-import com.google.api.resourcenames.ResourceNameType;
+import com.google.api.resourcenames.ResourceName;
 
 // AUTO-GENERATED DOCUMENTATION AND CLASS
 @javax.annotation.Generated("by GAPIC protoc plugin")
-public class BookNameType implements ResourceNameType {
+public abstract class BookNames {
```

Comment:
Why is this abstract?

Reply:
Do allow for the addition of new methods in the future without breaking implementations

Reply:
Implementations? Isn't this a container for static methods?

Reply:
Oh crap you're totally right, this shouldn't be abstract.